src/azure-cli/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -60,6 +62,19 @@ def load_arguments_eh(self, _):
         c.argument('key_version', is_preview=True,
                    help='The version of the KeyVault key to use.')
 
+# Cluster region
+    for scope in ['eventhubs cluster', 'eventhubs cluster namespace list']:
+        with self.argument_context(scope) as c:
+            c.argument('cluster_name', arg_type=name_type, id_part=None, help='Name of Cluster')
+
+    with self.argument_context('eventhubs cluster create') as c:
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), id_part=None, help='Location of the Cluster, for locations of available pre-provision clusters, please check az evetnhubs ')
+        c.argument('capacity', type=int, help='Capacity for Sku')
```

Comment:
Is there a range?

Reply:
allowed value is 1, in future might support more

Reply:
please add arg_type=get_enum_types[1]